transform/models/marts/fact_commits.sql

```diff
@@ -0,0 +1,6 @@
+SELECT 
+    DATE_TRUNC('day', event_date) AS date,
```

Comment:
Makes perfect sense to truncate, but we'll do that when we aggregate in the next layer of aggregated fact tables. For now, let's keep the date as is.